src/test/pybind/test_rbd.py

```diff
@@ -3077,6 +3077,15 @@ def test_group_image_list_move_to_trash(self):
         eq([], list(self.group.list_images()))
         RBD().trash_restore(ioctx, image_id, image_name)
 
+    def test_group_image_list_remove(self):
+        # need a closed image to get ImageMemberOfGroup instead of ImageBusy
```

Comment:
Is ImageBusy returned if the image is exclusively locked or not closed?

Reply:
In this context, if the image is not closed (or, more precisely, has a watcher).

Reply:
Is there were the EBUSY is raised, https://github.com/ceph/ceph/blob/v20.0.0/src/librbd/api/Trash.cc#L208 ?